### ISSUE TITLE
Is there a way to pass the username in the OAuth request URL and go directly to password page?

### ISSUE THREAD

**Issue by kmurugesan83** [MrSwitch/hello.js#467]
Hi,

I'm wondering if there is anyway I could bypass the username page and go directly to password page by passing the username in request query string params or something like that? for e.g. I have a xyz@gmail.com. Right now, it always takes me to the username page first and then go to password page. whereas once I logged in and logout, it remembers the username and takes me to password page directly. In fiddler they are two different URLs. so I'm wondering is it possible to achieve the same using hello.js?

Have anyone done anything like that before? Please let me know.

**Comment by MrSwitch** [comment]
Yeah i think google supports something like login_hints, e.g. `hello('google').login({login_hint: 'andrew@gmail.com'});` not sure if there's anything like this for other services, i'd be keen to know if there were.
